internal/proxy/proxy.go

```diff
@@ -29,11 +29,16 @@ import (
 )
 
 const (
-	GHHeader           = "User-Agent"
-	GHAgent            = "GitHub-Hookshot"
+	// GHHeader is how we determine whether we are dealing with a GitHub webhook
+	GHHeader = "User-Agent"
+	// GHAgent is the prefix of the GHHeader
+	GHAgent = "GitHub-Hookshot"
+	// CookieJenkinsIdled is name of the cookie through which we can check if service is idled
```

Comment:
This is not quite correct. I need to look at the code first, but this description is misleading.